Week3/homework/3-step3.js

```diff
@@ -2,24 +2,36 @@
 
 // use a 'for' loop
 function repeatStringNumTimesWithFor(str, num) {
-  // add your code here
-  return str;
-}
+    var reWhStr = "";
```

Comment:
Don't use `var`. Use `const` or `let` instead. Did you see [How to hand in your homework](https://github.com/HackYourFuture/JavaScript2/blob/master/Week3/MAKEME.md#how-to-hand-in-your-homework) in the MAKEME file?